exporter/importables.go

```diff
@@ -499,6 +506,25 @@ var resourcesMap map[string]importable = map[string]importable{
 							ID:       task.DbtTask.WarehouseId,
 						})
 					}
+					if task.DbtTask.Source == "WORKSPACE" {
+						directory := task.DbtTask.ProjectDirectory
+						if strings.HasPrefix(directory, "/Repos") {
+							ic.emitRepoByPath(directory)
+						} else {
+							nbAPI := workspace.NewNotebooksAPI(ic.Context, ic.Client)
+							objects, err := nbAPI.List(directory, true, true)
```

Comment:
Does the `true` mean here that it is recursive? A comment would be nice here as well (e.g., `// Traverse the dbt project directory and emit all objects found`)